src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault.java

```diff
@@ -1,87 +1,89 @@
-////////////////////////////////////////////////////////////////////////////////
-// Test case file for checkstyle.
-// Created: 2001
-////////////////////////////////////////////////////////////////////////////////
-package com.puppycrawl.tools.checkstyle.checks.blocks.emptyblock;
-
-import java.io.*; // star import for instantiation tests
-import java.awt.Dimension; // explicit import for instantiation tests
-import java.awt.Color;
-
-/**
- * Test case for detecting empty block statements.
- * @author Lars Kühne
- **/
-class InputEmptyBlockDefault
-{
-    static {
-        Boolean x = new Boolean(true);
+public class InputEmptyBlockDefault {
+    void method1(int a) {
```

Comment:
why are there so many changes in this file?
file should just be moved/copy. Changes should be minimal.